code/game/machinery/suit_storage_unit.dm

```diff
@@ -868,6 +868,18 @@
 	else if(href_list["apply_paintjob"])
 
 		if(!suit && !helmet) return
+		if(suit.helmet)
+			to_chat(usr, "<span class='alert'>\The [src] cannot function while a helmet is attached to the suit!</span>")
+			return
+		var/list/no_refit
+		if (helmet && !helmet.refittable)
+			LAZYADD(no_refit, helmet)
+		if (suit && !suit.refittable)
+			LAZYADD(no_refit, suit)
+		if (LAZYLEN(no_refit))
+			to_chat(usr, "<span class='alert'>\The [english_list(no_refit)] in [src] is not refittable!</span>")

```

Comment:
Wonder if it'd be worth handling `is/are` properly. Perhaps instead of `is`, `[no_refit.len == 1 ? "is" : "are"]`?